apple-pie/styles/style.css

```
@@ -5,6 +5,120 @@ html {
   font-size: 16px;
 }
 
+.ApplePiePhoto {
+  width: 850px;
+  height: 550px;
+}
+
+.containerApplePiePhoto {
+  background-image: url("images/apple-pie.jpg");
```

```suggestion
  background-image: url(../images/apple-pie.jpg);
```
En este caso si te fijas el css esta dentro de la carpeta styles, y dentro de styles no esta images, por ello salimos una carpeta con el .., y ahora ya si podiamos acceder a images :D

---

¡De acuerdo, Cristian! Estar más atenta a poner bien las URLs ^^